components/select/nz-option-li.spec.ts

```diff
@@ -50,19 +50,6 @@ describe('select option li', () => {
       fixture.detectChanges();
       expect(liComponent.selected).toBe(false);
     });
-    it('should active work', () => {
```

Comment:
test needed

Reply:
varible `active` is not neccessary. So I delete the  this test func.